.github/workflows/npm-publish.yml

```diff
@@ -17,7 +21,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: ${{ matrix.node }}
```

Comment:
In my opinion we should only be publishing the package with the stable node version.

Reply:
yes, we should have only one version of Node publishing this package, i.e 12, later we can drop older versions and set it to 16